.circleci/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# This image is used by CircleCI to run tests.
+# It's published as freelawproject/courtlistener-testing:x.y.z.
```

Comment:
@mlissner, I created a [freelawproject org on docker hub](https://hub.docker.com/u/freelawproject/dashboard/) where this image is uploaded to. If you create an account there, I can add you to the freelawproject team that owns this hub org.